eth-reference-tests/src/referenceTest/java/tech/pegasys/teku/reference/ManualReferenceTestRunner.java

```diff
@@ -30,7 +29,7 @@
  * <p>The test case is disabled as the tests run via the generated classes in CI, but it still runs
  * without removing the @Disabled in IntelliJ.
  */
-@Disabled
+// @Disabled
```

Comment:
We should disable this again before merge so it doesn't run in CI.

Reply:
Ups, thanks 👍